src/green/troubleshooting/resetting-the-device-using-sd-card.md

```diff
@@ -25,86 +32,134 @@ Perform this procedure if you want to perform a factory reset on your device.
 
 ## To copy the Home Assistant Green OS to the SD card
 
-1. To download the Home Assistant Green OS installer, paste this [URL](https://github.com/NabuCasa/buildroot-installer/releases/tag/green-installer-20240410) into your browser.
-   - To start the download, select the file *green-installer-20240410.img.xz*.
-   - Download and start [Balena Etcher](https://www.balena.io/etcher).
-      - You may need to run it with administrator privileges on Windows.
-   ![Downloading the required software](/static/img/green/download-circle-outline.svg)
+{% steps %}
+{% step "Downloading the Home Assistant Green OS installer" %}
+{% image "/static/img/green/download-circle-outline.svg" "Downloading the required software" %}
+{% stepContent %}
+
+- To download the Home Assistant Green OS installer, paste this [URL](https://github.com/NabuCasa/buildroot-installer/releases/tag/green-installer-20240410) into your browser.
```

Comment:
This version numbering should be:
a) automatic
b) a variable
c) both a and b
(but out of scope here)

Reply:
Which version number are you referring to?